test/extended/networking.sh

```diff
@@ -7,7 +7,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-if [[ "${OPENSHIFT_QUIET_OUTPUT:-false}" != "true" ]]; then
+if [[ "${OPENSHIFT_QUIET_OUTPUT:-true}" != "true" ]]; then
```

Comment:
I'm fine with this being the default for non-ci, but having as much output as possible when a job fails is generally useful.  Maybe default to false only if OPENSHIFT_IMAGE_REGISTRY is not provided, since that's now provided by default in ci?